admin/hotlist-service/src/main/java/io/mosip/hotlist/config/HotlistConfig.java

```diff
@@ -102,7 +101,7 @@ private Map<String, Object> additionalProperties() {
 		jpaProperties.put("hibernate.dialect", dialect);
 		jpaProperties.put("hibernate.temp.use_jdbc_metadata_defaults", Boolean.FALSE);
 		jpaProperties.put("hibernate.implicit_naming_strategy", SpringImplicitNamingStrategy.class.getName());
-		jpaProperties.put("hibernate.physical_naming_strategy", SpringPhysicalNamingStrategy.class.getName());
+		jpaProperties.put("hibernate.physical_naming_strategy", CamelCaseToUnderscoresNamingStrategy.class.getName());
```

Comment:
Is it tested by running and the DB updates working fine locally?

Reply:
tested locally, working fine as expected.